model/src/main/scala/hmda/model/census/CBSATractLookup.scala

```diff
@@ -6,7 +6,9 @@ import scala.io.Source
 
 object CBSATractLookup {
   val values: Seq[CBSATract] = {
-    val lines = Source.fromFile(new File("model/src/main/resources/tract_to_cbsa_2013.csv")).getLines
+    val cbsaResource = getClass.getResourceAsStream("/tract_to_cbsa_2013.csv")
+    val lines = Source.fromInputStream(cbsaResource).getLines()
```

Comment:
I'm torn: I notice that these lines are exactly like lines 9-10 in the other file (apart from the file name).  And so I want to suggest extracting a method and putting it into some convenient utility place, called `ResourceUtils` or some such bundle-of-things name.
But then there might have to be a `ResourceUtils` for each of `model`, `api`, etc. Which feels weird.

I still lean toward extracting it somewhere.  For one thing, maybe only `model` will even have such resources!  If the imagined situation becomes a problem, we can deal with it then.